app/models/katello/subscription.rb

```diff
@@ -33,6 +37,14 @@ def recently_expired?
       pools.any?(&:recently_expired?)
     end
 
+    def virt_who
```

Comment:
i'd probably propose something like 'using_virt_who' for the class method, and 'virt_who_pools' for the subscription method (or something similar)